packages/story-editor/src/components/form/linkIcon.js

```diff
@@ -36,6 +37,7 @@ const StyledMedia = styled(Media)`
 `;
 
 function LinkIcon({ handleChange, icon, isLoading = false, ...rest }) {
+  const enablePosterHotlinking = useFeature('posterHotlinking');
```

Comment:
@swissspidy Do we need a new feature flag here as well?

Reply:
Yes, I prefer separate feature flags for all of these so that we can roll them out separately if needed.